jdaviz/configs/imviz/helper.py

```diff
@@ -174,37 +214,88 @@ def load_static_regions(self, regions, **kwargs):
             Extra keywords to be passed into the region's ``to_mask`` method.
             This is ignored if Numpy array is given.
 
+        Returns
+        -------
+        bad_regions : dict
+            Dictionary mapping region names to the regions that failed to load.
+            If all the regions loaded successfully, this will be empty.
+
         """
+        bad_regions = {}
         # Subset is global, so we just use default viewer.
         data = self.default_viewer.state.reference_data
 
+        # TODO: Enable this after https://github.com/glue-viz/glue/issues/2275 is fixed.
+        # existing_labels = [lyr.layer.label for lyr in self.default_viewer.layers]
```

Comment:
This check is blocked by:

* glue-viz/glue#2275

Basically region that failed to load because it is out of bounds will be registered in the viewer layers anyway. But if it fails to load because the region object is not supported, then it is not a problem.